spec/setup_test_database.rb

```diff
@@ -0,0 +1,6 @@
+require 'pg'
+
+def setup_test_database
+  connection = PG.connect(dbname: 'chitter_test', user: 'postgres', password: 'postgres')
```

Comment:
the inclusion of **user**: and **password**: meant that I couldn't actually run rspec, either. I had to replace the connect statement with this, for it to work.
  
**connection = PG.connect(dbname: 'chitter_test')**